results/2021/nullified_polling_stations.csv

```diff
@@ -0,0 +1,124 @@
+
+,District,Constituency,Scounty,Parish,,POLLING STATION,VOTERS
+No.,,,,,,,
+1,32 MUKONO,231 MUKONO MUNICIPALITY,01 GOMA DIVISION,05 SEETA WARD,31 GOSHEN LAND [NAK-Z],,823
+2,32 MUKONO,176 MUKONO COUNTY NORTH,02 KYAMPISI,14 KYABAKADDE,08 KASALA,,412
+3,32 MUKONO,176 MUKONO COUNTY NORTH,02 KYAMPISI,16 NTONTO,05 KASENENE,,419
+4,32 MUKONO,176 MUKONO COUNTY NORTH,04 NAMA,20 NAMAWOJJOLO,07 NAMAWOJJOLO ISLAMIC P/S [N-Z],,933
+5,32 MUKONO,176 MUKONO COUNTY NORTH,04 NAMA,20 NAMAWOJJOLO,08 NAMAWOJJOLO WEST [N-Z],,757
+,,062 KAWEMPE DIVISION,,,,,
```

Comment:
@morelmiles This is an example of a duplicate record. This file should have 111 lines (including the header)